lib/fastlane/plugin/run_tests_firebase_testlab/actions/run_tests_firebase_testlab_action.rb

```diff
@@ -192,6 +214,67 @@ def self.category
         :testing
       end
 
```

Comment:
How should we refactor this?

Reply:
I think the best way is to separate it by modules. We need to create an option module that will contain all the code related to the available options and other helpers so we can delegate the code instead of having everything inside the action module.